src/engraving/libmscore/edit.cpp

```diff
@@ -1959,7 +1959,7 @@ void Score::addHairpin(HairpinType type)
         for (int staffIdx = selection().staffStart(); staffIdx < selection().staffEnd(); ++staffIdx) {
             ChordRest* cr1 = selection().firstChordRest(staffIdx * VOICES);
             ChordRest* cr2 = selection().lastChordRest(staffIdx * VOICES);
-            addHairpin(type, cr1, cr2, /* toCr2End */ true);
+            return addHairpin(type, cr1, cr2, /* toCr2End */ true);
```

Comment:
Wouldn't that leave the function in the first round of that `for` loop? The comment says `// add hairpin on each staff if possible`

Reply:
Thanks, I didn't notice this for loop :(

Reply:
fixed